cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -0,0 +1,172 @@
+const NINTH_ELEMENT = 8;
+
+Then('FlatTable has sticky row header', () => {
+  const color = 'rgb(204, 214, 218)';
+  flatTableRowHeader(SECOND_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(THIRD_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(FOURTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(FIFTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(SIXTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(SEVENTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(EIGHTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableRowHeader(NINTH_ELEMENT).should('have.css', 'border-right-color', color)
+    .and('have.css', 'border-left-color', color)
+    .and('have.css', 'border-bottom-color', color)
+    .and('have.css', 'position', 'sticky')
+    .and('be.visible');
+});
+
+Then('FlatTable has sticky header', () => {
+  flatTableHeader(SECOND_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(THIRD_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(FOURTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(FIFTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(SIXTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(SEVENTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+  flatTableHeader(EIGHTH_ELEMENT).should('have.css', 'position', 'sticky')
+    .and('be.visible');
+});
+
+Then('FlatTable has nine row(s)', () => {
```

Comment:
we should precise that FlatTable has rows not `row(s)`

Reply:
fixed